co-authors-plus.php

```diff
@@ -1289,14 +1291,16 @@ function filter_views( $views ) {
 				'author_name'           => wp_get_current_user()->user_nicename,
 			);
 		if ( 'post' != get_post_type() ) {
-			$mine_args['post_type'] = get_current_screen()->post_type;
+			$mine_args['post_type'] = get_post_type();
```

Comment:
We can probably get rid of this entire conditional block if we are getting the post type up as per ^^.